config/s3/workloads/s3_copy_object.yaml

```diff
@@ -0,0 +1,119 @@
+test_1:
```

Comment:
Workload directory structure should be tool specific for ex s3api: config/s3/s3api/test.yaml

Reply:
Done